src/VisualStudio/Core/Def/Implementation/Library/ObjectBrowser/AbstractObjectBrowserLibraryManager.cs

```diff
@@ -76,7 +76,9 @@ public void Dispose()
                     var oldDocument = e.OldSolution.GetDocument(e.DocumentId);
                     var newDocument = e.NewSolution.GetDocument(e.DocumentId);
 
-                    await DocumentChangedAsync(oldDocument, newDocument).ConfigureAwait(false);
+                    // make sure we do this in background thread. we don't care about ordering of events
+                    // we just need to refresh OB at some point if it ever needs to be updated
+                    Task.Run(() => DocumentChangedAsync(oldDocument, newDocument));
```

Comment:
Note: there's a comment that says:

                    // Ensure the text version actually changed. This is necessary to not
                    // cause Class View to update simply when a document is opened.

However, i'm curious why we would care about that that much.  Say someone opens a document.  Is it that bad if OB then updates in some tab they're not looking at?